README.md

```diff
@@ -183,21 +154,5 @@ i18n({
 });
 ```
 
-Resources returned by the callback will be an object of the following form:
 
-```javascript
-{
-    en: {
-        ... translation keys for default namespace ...
-        'other-namespace': {
-            ... translation keys for afternative namespace...
-        }
-    },
-    fr: {
-        ...
-    },
-    de: {
-        ...
-    }
-}
-```
+*Note: if you are using hmpo-i18n inside a module which is likely to be installed as a child dependency of another project then it is highly recommended that you set this property.*
```

Comment:
This is to do with the baseDir option, one of the things lost when the context was removed from some options